index.html

```diff
@@ -237,6 +237,76 @@ <h4 class="project-title roboto white">Multi-Post Stories</h4>
           </div>
         </div>
       </section>
+      <section class="contact-form flex-container-column" id="contact">
+        <div>
+          <h2 class="roboto head-color align-center">CONTACT US</h2>
+          <p class="inter white align-center">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I'd love
+            to help with it
+          </p>
+        </div>
+        <form
+          action="https://formspree.io/f/mrgjweev"
+          method="post"
+          class="flex flex-container-column"
+        >
+          <input
+            type="text"
+            name="full-name"
+            id="full-name"
+            class="input-cls inter text-400"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            name="email"
+            id="email"
+            class="input-cls inter text-400"
+            placeholder="name@gmail.com"
+            required
+          />
+          <textarea
+            class="input-cls message-box inter text-400"
+            name="message"
+            id="message"
+            cols="45"
+            rows="8"
+            maxlength="500"
+            placeholder="Enter text here"
+            required
+          ></textarea>
+          <input
+            type="submit"
+            class="btn btn-primary roboto white submit-btn"
+            value="Submit"
+          />
+        </form>
+      </section>
     </main>
```

Comment:
- [OPTIONAL] There is an anomaly overflow on the x-axis when the project is inspected/simulated to mobile view. For future references, kindly consider working on that 👍🏽 .